hc/api/management/commands/sendalerts.py

```diff
@@ -21,8 +21,10 @@ def handle_many(self):
         now = timezone.now()
         going_down = query.filter(alert_after__lt=now, status="up")
         going_up = query.filter(alert_after__gt=now, status="down")
+        nag_down = query.filter(nag_after__lt=now, status="down")
+        nag_up = query.filter(nag_after__gt=now, status="nag")
```

Comment:
We can add nag functionality for my story on jobs running too often after the demo.

Reply:
Good implementation of nag slider